web/docs/url-testing/start.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#
+# This script handles starting up a single (i.e. non-stack)
+# NGINX container to test redirects to an external web service.
+#
+
+# Run the system
+echo "Building and starting containers..."
+docker compose -f ./docker-compose.yml up -d --build
```

Comment:
```suggestion
docker compose -f ./docker-compose.yml up -d --build || exit 1
```

Exit if the docker compose fails